axiom/monitors.go

```diff
@@ -145,6 +148,20 @@ type Monitor struct {
 	Disabled bool `json:"disabled"`
 	// DisabledUntil is the time that the monitor will be disabled until.
 	DisabledUntil time.Time `json:"disabledUntil"`
+	// SecondDelay is the delay in seconds after the end time that the monitor runs.
+	SecondDelay int64 `json:"secondDelay,omitempty"`
```

Comment:
Can you use a `time.Duration`, here? See how it's done for `Interval`, it is specifically marshalled/unmarshalled. I think it is the same for `Range`.

Couple of notes:

1. Add a comment saying the lowest resolution is seconds (and please do me a favour and add the same comment to `Interval` explaining that the lowest resolution is minutes)
2. Why is it that `secondDelay` has a lowercase `s` but `IntervalMinutes` has an uppercase `I`?